.github/workflows/e2eNew.yml

```diff
@@ -45,6 +30,16 @@ on:
         required: false
         default: true
         type: boolean
+      lwcLsp:
+        description: 'LWC LSP'
+        required: false
+        default: true
+        type: boolean
+      visualforceLsp:
```

Comment:
I realized just now that the LSP E2E tests currently contain only Aura and Visualforce.  Given that finding, we can just change that workflow to run on ESM (the same change made to Apex and Core) and remove those two from this ESM test suite.

Reply:
So this ESM test suite would contain:
1. Manifest Builder
2. Push and Pull
3. LWC LSP